src/python/knnPerfTest.py

```diff
@@ -102,9 +90,11 @@ def advance(ix, values):
 
 
 def run_knn_benchmark(checkout, values):
-  indexes = [0] * len(values.keys())
-  indexes[-1] = -1
-  args = []
+  DO_PROFILING = values.pop("profile")
```

Comment:
Hmm this used to be global scope, does it matter that it's now local to this method?

Reply:
OK that's great -- moving a variable from global to local scope when it's already only used in that one local scope is rote refactoring.

> I just realized that I forgot to change the naming given that these arguments are no longer constants too.

Ahh you mean the ALL_CAPS styling?  Yeah let's fix that in another rev?